code/modules/mob/living/basic/space_fauna/bear/bear.dm

```diff
@@ -147,23 +153,20 @@
 	attack_verb_simple = "slap"
 	attack_verb_continuous = "slaps"
 
-/mob/living/simple_animal/hostile/bear/butter/add_cell_sample()
-	return //You cannot grow a real bear from butter.
-
-/mob/living/simple_animal/hostile/bear/butter/Life(seconds_per_tick = SSMOBS_DT, times_fired) //Heals butter bear really fast when he takes damage.
+/mob/living/basic/bear/butter/Life(seconds_per_tick = SSMOBS_DT, times_fired) //Heals butter bear really fast when he takes damage.
 	if(stat)
 		return
 	if(health < maxHealth)
 		heal_overall_damage(5 * seconds_per_tick) //Fast life regen, makes it hard for you to get eaten to death.
```

Comment:
overriding life entirely seems really fucking wacky and bugprone, could we listen for `COMSIG_LIVING_HEALTH_UPDATE` to trigger this instead? It's just really spooky that we were _never_ invoking parent here.

also treating `stat` as a boolean is cringe and it would be greatly appreciated if you could make it something very explicit like `(stat == DEAD)` or whatever it is